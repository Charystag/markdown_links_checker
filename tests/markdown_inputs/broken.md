# Test Case: Broken HTTP/HTTPS URLs (No Parentheses)

This test case includes broken HTTP/HTTPS URLs that should return a `404 Not Found` or another error status.

## Example 1: Nonexistent Domains

- [Nonexistent Domain HTTP](http://thisdomaindoesnotexist123456.com)
- [Nonexistent Domain HTTPS](https://thisdomaindoesnotexist123456.com)

## Example 2: Valid Domain with Invalid Path

- [Google Invalid Path](https://www.google.com/thispagedoesnotexist)
- [GitHub Invalid Path](https://github.com/thispagedoesnotexist)

## Example 3: Typo in Domain Name

- [Typo in Domain](http://wwww.example.com)  <!-- Note the extra 'w' -->
- [Typo in Domain HTTPS](https://githbusss.com)

## Example 4: Invalid Top-Level Domains (TLDs)

- [Invalid TLD](http://example.invalidtld)
- [Another Invalid TLD](https://website.nonexistenttld)

## Example 5: URLs with Correct Domain but Not Found

- [Wikipedia Nonexistent Page](https://en.wikipedia.org/wiki/This_Page_Does_Not_Exist)
- [BBC Nonexistent Article](https://www.bbc.com/thisarticleisnotfound)

## Example 6: URLs in Different Markdown Contexts

### 6.1: URL in a Table

| Description        | Link                                               |
|--------------------|----------------------------------------------------|
| Broken Table URL   | [Broken Table URL](https://example.com/brokenpath)  |

### 6.2: URL in a List

- Broken URL in a list: [Broken List URL](https://example.com/brokenlist)

### 6.3: URL in a Blockquote

> Here is a broken URL in a blockquote: [Broken Blockquote URL](https://example.com/brokenquote)

## Example 7: Redundant Reference Links (Broken)

Here is a broken redundant reference link:

[Broken Redundant Link][1]

[1]: https://example.com/brokenredundant

## Example 8: Mixed Content with Broken Links

Text before the broken link [Broken Mixed Content](https://example.com/broken_mixed) and text after the link.
