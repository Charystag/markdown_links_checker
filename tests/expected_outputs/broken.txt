Checking file: `broken.md'
[0;31mError on line : `7'. url: `http://thisdomaindoesnotexist123456.com'. response_code: `000'[0m
[0;31mError on line : `8'. url: `https://thisdomaindoesnotexist123456.com'. response_code: `000'[0m
[0;31mError on line : `12'. url: `https://www.google.com/thispagedoesnotexist'. response_code: `404'[0m
[0;31mError on line : `13'. url: `https://github.com/thispagedoesnotexist'. response_code: `404'[0m
[0;31mError on line : `17'. url: `http://wwww.example.com'. response_code: `000'[0m
[0;31mError on line : `18'. url: `https://githbusss.com'. response_code: `000'[0m
[0;31mError on line : `22'. url: `http://example.invalidtld'. response_code: `000'[0m
[0;31mError on line : `23'. url: `https://website.nonexistenttld'. response_code: `000'[0m
[0;31mError on line : `27'. url: `https://en.wikipedia.org/wiki/This_Page_Does_Not_Exist'. response_code: `404'[0m
[0;31mError on line : `28'. url: `https://www.bbc.com/thisarticleisnotfound'. response_code: `404'[0m
[0;31mError on line : `36'. url: `https://example.com/brokenpath'. response_code: `500'[0m
[0;31mError on line : `40'. url: `https://example.com/brokenlist'. response_code: `500'[0m
[0;31mError on line : `44'. url: `https://example.com/brokenquote'. response_code: `500'[0m
[0;31mError on line : `52'. url: `https://example.com/brokenredundant'. response_code: `500'[0m
[0;31mError on line : `56'. url: `https://example.com/broken_mixed'. response_code: `500'[0m
[0;31mbroken links: 15/15[0m
